source/includes/_opportunities.md

```diff
@@ -81,7 +81,7 @@ print(response.text)
 | closed_date          | The date the Opportunity was closed                                                                                                         |
```

Comment:
Line no 78 
`Opportunity Name` -> `Opportunity name` 

Line no 79
`The Amount of money involved in the Opportunity/ Deal` -> `The amount of money involved in the opportunity/ deal`

Line no 80 
`The ID of the current Stage .You can get a list of all opportunity stages from Misc/Opportunity Stages` -> `The ID of the current stage .You can get a list of all opportunity stages from misc/opportunity stages` 

Line no 81
`The date the Opportunity was closed ` -> `The date the opportunity was closed `

Line no 82
`ID of the Account ` -> `ID of the account`